TLM/TLM/UI/Helpers/SerializableUIOptionBase.cs

```diff
@@ -64,10 +66,11 @@ public abstract class SerializableUIOptionBase<TVal, TUI, TComponent> : ILegacyS
 
         /// <summary>Gets or sets the value of the field this option represents.</summary>
         public virtual TVal Value {
-            get => _fieldInfo != null ? (TVal)_fieldInfo.GetValue(null) : _value;
+            get => _fieldInfo != null ? (TVal)Convert.ChangeType(_fieldInfo.GetValue(null), typeof(TVal)) : _value;
```

Comment:
maybe `Type.GetTypeCode()` could be be more efficient in avoiding heap allocation?